lib/travis/build/script/r.rb

```diff
@@ -84,7 +84,9 @@ def setup
             sh.cmd 'sudo chmod 2777 /usr/local/lib/R /usr/local/lib/R/site-library'
 
           when 'osx'
-            sh.cmd 'brew update', retry: true
+            # We want to update, but we don't need the 800+ lines of
+            # output.
+            sh.cmd 'brew update >/dev/null', retry: true
```

Comment:
You can also use `echo: false` instead. Either is fine; I'm just pointing it out.